examples/libsodium/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.0)
+project(libsodium-example)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
```

Comment:
This one need to be placed before `project`. For now all tests failed because of this error.